### PR TITLE
Namadillo: Fees

### PR DESCRIPTION
**Pull request by pedrorezende**
This PR adds back the fees panel with its respective warning

### CODE REVIEW

File: apps/namadillo/src/atoms/fees/services.ts [hunk 1]
```diff
@@ -30,7 +30,8 @@ export const fetchMinimumGasPrice = async (
     ({ token }) => token === nativeToken
   );
   invariant(!!nativeTokenCost, "Error querying minimum gas price");
-  const asBigNumber = new BigNumber(nativeTokenCost.amount);
+  // TODO: this should be removed after indexer error is fixed!
```
Comment (emccorson) [inline apps/namadillo/src/atoms/fees/services.ts:33]: It looks like this is fixed now.

File: apps/namadillo/src/App/Common/TransactionFees.tsx [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
-        forceBalanceDisplay={true}
-      />
+    <div className={clsx("flex flex-col", className)}>
+      <div className="text-white text-sm">
+        Transaction fee:{" "}
+        <NamCurrency
+          className="font-medium"
+          forceBalanceDisplay={true}
+          amount={gasPrice.data.multipliedBy(
+            gasLimits.data[txKind].native.multipliedBy(numberOfTransactions)
```
Comment (emccorson) [inline apps/namadillo/src/App/Common/TransactionFees.tsx:35]: Will this still be correct in cases where a reveal PK is needed?

**Review by emccorson — COMMENTED** [review]
Looks good! I just a left of couple of comments and I would like to quickly check this again after the merge conflicts are fixed.

File: apps/namadillo/src/App/Common/TransactionFees.tsx (hunk 2 above)
Comment (pedrorezende) [reply apps/namadillo/src/App/Common/TransactionFees.tsx:35]: I think I need to review it again after the batch tx PR gets in

Comment (jurevans) [reply apps/namadillo/src/App/Common/TransactionFees.tsx:35]: With batch Tx in, RevealPk wasn't considered when calculating fees - it is overestimating though so will work as is (though is incorrect). We still need to determine how fees can be calculated when we have a batch with of Txs, and (currently) at most 2 Tx types in that batch (RevealPK + the type of other Txs). I'm up for taking a look, and we don't necessarily need to address that in this PR!

**Review by jurevans — APPROVED** [review]
LGTM! Left a comment about fee estimation - that can be handled separately, as it needs a bit of investigation and discussion!

**Comment by pedrorezende** [comment]
Closing it for now

**Pull request closed without merging**